src/commands/completion/index.ts

```diff
@@ -4,7 +4,7 @@ import { getInstructionsForFish } from '../../utils/fish'
 import { getInstructionsForZsh } from '../../utils/zsh'
 
 export default class Completion extends CompletionBase {
-  static description = 'completion plugin'
+  static description = 'Add shell completion'
```

Comment:
```suggestion
  static description = 'Genetate shell completion script'
```

We should use the word "Generate" instead of "Add" 😃

Reply:
yep, that's better :)